pages/common/nm.md

```diff
@@ -0,0 +1,19 @@
+# nm
+
+> List symbol names in object files
+
+- List global (extern) functions in a file (prefixed with T)
+
+`nm -g {{file.o}}`
+
+- Demangle C++ symbols (make them readable)
+
+`nm {{file.o}} | c++filt`
```

Comment:
Unfortunately, this is the only cross-platform way (between GNU, BSD) of doing C++ name demangling (GNU does have the [`--demangle` option](http://man7.org/linux/man-pages/man1/nm.1.html#OPTIONS)). Since `nm` output is less than helpful for C++ object files without name demangling (and, personally, the whole reason I'd use a tldr page for `nm` anyway), `c++filt` should be mentioned in here somewhere. Should it be put in a "see also" in the description?